File: platforms/ios/Dezel/Dezel/Modules/Graphic/GraphicModule.swift
/**
 * @class GraphicModule
 * @since 0.1.0
 * @hidden
 */
open class GraphicModule: Module {

	//--------------------------------------------------------------------------
	// MARK: Methods
	//--------------------------------------------------------------------------

	/**
	 * @inherited
	 * @method initialize
	 * @since 0.1.0
	 */
	override open func initialize() {
		self.context.registerClass("dezel.graphic.Image", type: Image.self)
		self.context.registerClass("dezel.graphic.Canvas", type: Canvas.self)
	}
}

File: platforms/android/Dezel/dezel/src/main/java/ca/logaritm/dezel/modules/graphic/GraphicModule.kt
package ca.logaritm.dezel.modules.graphic

import ca.logaritm.dezel.core.JavaScriptContext
import ca.logaritm.dezel.core.JavaScriptModule

/**
 * @class GraphicModule
 * @since 0.1.0
 * @hidden
 */
open class GraphicModule(context: JavaScriptContext): JavaScriptModule(context) {

	//--------------------------------------------------------------------------
	// Methods
	//--------------------------------------------------------------------------

	/**
	 * @method initialize
	 * @since 0.1.0
	 */
	override fun initialize() {
		this.context.registerClass("dezel.graphic.Image", JavaScriptImage::class.java)
		this.context.registerClass("dezel.graphic.Canvas", JavaScriptCanvas::class.java)
	}
}
